lib/commons/color/get-background-color.js

```diff
@@ -25,6 +26,34 @@ export default function getBackgroundColor(
   bgElms = [],
   shadowOutlineEmMax = 0.1
 ) {
+  const vNode = getNodeFromTree(elm);
+  const bgColorCache = vNode._cache.getBackgroundColor;
+
+  if (bgColorCache) {
+    bgElms.push(...bgColorCache.bgElms);
+    incompleteData.set('bgColor', bgColorCache.incompleteData);
+    return bgColorCache.bgColor;
+  }
+
+  const bgColor = _getBackgroundColor(elm, bgElms, shadowOutlineEmMax);
+
+  // TODO: this is a temporary solution to the problem
+  // of needing to cache the getBackgroundColor call
+  // but not being able to use `memoize` since this
+  // is not a pure function due to the `bgElms` array
+  // and `incompleteData`.
+  // remove after 4.5 release when we have time to
+  // properly update the function to be more cacheable
```

Comment:
Please put this in an issue instead.

Reply:
... No, I mean take this out of the code and put it into an issue.

Reply:
https://github.com/dequelabs/axe-core/issues/3721